Formula/stone.rb

```diff
@@ -15,7 +15,7 @@ class Stone < Formula
   option "with-ssl", "SSL support"
 
   def install
-    target = (build.with? "ssl") ? "macosx-ssl" : "macosx"
+    target = build.with?("ssl") ? "macosx-ssl" : "macosx"
```

Comment:
Wouldn't need to pull bottles here given it's optional, but happy to file a PR, aye.